.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,58 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'master'
+            - '[0-9]+.x'
+            - '[0-9]+.[0-9]+'
+            - '[0-9]+.[0-9]+.x'
+
+jobs:
+    test:
+        name: 'PHP ${{ matrix.php-version }}, Symfony ${{ matrix.symfony-version }}'
+        runs-on: ubuntu-20.04
+
+        env:
+            SYMFONY_DEPRECATIONS_HELPER: ${{ matrix.symfony-deprecation-helper }}
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '7.1'
+                      composer-flags: '--prefer-lowest'
+                      symfony-version: '^3.4'
+                      symfony-deprecation-helper: 'weak'
+
+                    - php-version: '7.2'
+                    - php-version: '7.3'
+                    - php-version: '7.4'
+                      symfony-version: '^4.4'
+                    - php-version: '7.4'
+                      symfony-version: '5.0.*'
+                    - php-version: '8.0'
+
+        steps:
+            - name: Checkout project
+              uses: actions/checkout@v2
+
+            - name: Install and configure PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-version }}
+                  tools: 'composer:v2'
+
+            - name: Require Specific Symfony Version
+              if: ${{ matrix.symfony-version }}
+              run: composer require --no-update symfony/symfony:${{ matrix.symfony-version }}
```

Comment:
can we please use flex here, like we did with travis? that is a bit more reliable as requiring symfony/symfony can hide dependencies.

Reply:
Thats a really good idea 👍 did update it!